sdk/containerregistry/azure-containerregistry/README.md

```diff
@@ -137,6 +143,105 @@ with ContainerRegistryClient(endpoint, DefaultAzureCredential(), audience="https
                 client.delete_manifest(repository, manifest.digest)
 ```
 
+### Upload images
+
+To upload a full image, we need to upload individual layers and configuration. After that we can upload a manifest which describes an image or artifact and assign it a tag.
+
```

Comment:
Can you add the snippet annotation like : \<!-- SNIPPET:sample_authentication.authentication_client_with_metrics_advisor_credential -->?

Reply:
Updated